timeoff/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from datetime import datetime, date
+from django.utils import timezone
+from accounts.models import User
+
+
+# Create your models here.
+class Timeoff(models.Model):
+    id = models.AutoField(primary_key=True)
+    username = models.ForeignKey(User, on_delete=models.CASCADE)
+    requested_timeoff = models.DateField(auto_now=False)
+    text_box = models.TextField(max_length=500)
+    submission_date = models.DateTimeField('date submitted',default=timezone.now)
+
+
+
+class Status(models.Model):
+    STATUS_CHOICES = [ ('A','Approved'),('D','Denied'),('P','Pending')]
+    id = models.OneToOneField('Timeoff', on_delete=models.CASCADE, primary_key=True)
```

Comment:
you can change this field to timeoff or something like that to indicate that it is a foreign key. You can remove the primary key attribute and let django handle the ids automatically like above.